smartseq2_single_sample/run_multi_samples/ss2_multi_sample_wf.wdl

```diff
@@ -0,0 +1,98 @@
+import "ss2_single_sample_wf.wdl" as singlesample
+
+task GatherMetrics {
+
+  Array[File]+ input_metrics_fn
+  String output_filename
+   
+  command <<<
+    # To parse out Picard metrics
+    i=1
+    for f in ${sep=' ' input_metrics_fn};do
+      sid=$(echo $f|rev| cut -d '/' -f 1 | rev |cut -d '.' -f 1)
+      if [ $i == "1" ];then
+        # print out header if i ==1, otherwise not. also attach a sampleID at each beginning of line
+        header=`cat $f|awk 'NR==7 {print $_}'`
+        echo "SampleID "$header > "${output_filename}_metrics"
+        i=0
+      fi
+      # Most picard metrics starts at 8th row, except aln metrics.the 8th and 9th row of aln metrics is the 
+      # R1 and R2 metrics, 10th row is the metrics for pair reads 
+      if [ ${output_filename} == "aln" ];then
```

Comment:
Do these metrics files use # or something to indicate header rows? If so we could pipe through grep -v instead of having this if else block. grep -v seems like it would be a bit more robust.

Reply:
Picad does not output files this way.

Reply:
I don't think picard does this way.